lib/installer.js

```diff
@@ -10,6 +10,7 @@ const WAD_VER = '1.2-RC';
 const WAD_DOWNLOAD_LINK = `https://github.com/Microsoft/WinAppDriver/releases/download/v${WAD_VER}/WindowsApplicationDriver.msi`;
 const WAD_DOWNLOAD_MD5 = 'dbaa9a3f7416c2b73cc5cd0e7452c8d0';
 const WAD_DOWNLOAD_TIMEOUT_MS = 60000;
+const WAD_INSTALLED_PATH = process.env.WAD_PATH;
```

Comment:
There is no need for a static variable there.

Reply:
Fixed.